### ISSUE TITLE
some question about the model

### ISSUE THREAD

**Issue by Carbord** [Joker316701882/Additive-Margin-Softmax#23]
Hi, I want to know something more about the News on 2018/08/29/.
You mentioned that "latest experiment-- Resface20(bn) + vggface2 + weight_decay5e-4 + batch_size256 + momentum achieves 0.995+-0.003". I think Resface20 represents the model you used. But what does the "vggface2" means here? Does it means a pre-trained model ?
Gratitude for your reply soon, thanks~

**Comment by Joker316701882** [comment]
@Carbord 
Vggface2 is a public available dataset. see details here:
https://arxiv.org/abs/1710.08092
